support-frontend/assets/components/subscriptionCheckouts/address/postcodeFinder.jsx

```diff
@@ -18,8 +16,21 @@ import {
 import { type AddressType } from 'helpers/subscriptionsForms/addressType';
 import { type PostcodeFinderResult } from 'components/subscriptionCheckouts/address/postcodeLookup';
 
-import styles
-  from 'components/subscriptionCheckouts/address/postcodeFinder.module.scss';
+// Styles
```

Comment:
Nitpick but I wouldn't bother with this comment unless we're going to do it everywhere which we don't.

Reply:
Sure. It's the pattern in this file - all sections have a comment in this style. However, I am not attached to it so happy to remove.